examples/chip-tool/main.cpp

```diff
@@ -36,6 +37,7 @@ int main(int argc, char * argv[])
 
     registerCommandsEcho(commands);
     registerClusters(commands);
+    registerCommandsParse(commands);
```

Comment:
nit: Move this initialisation before `registerClusters`. That's not a big deal but I would like to keep the commands that are not really _clusters_ close together.

Reply:
Good point, resolved in the new revision.